File: Lv.1/푸드 파이트 대회.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(vector<int> food) {
    
    string answer = "";
    string first = "";
    for(int i=1;i<food.size();i++){
        
        int temp = food[i]/2;
        for(int j=0;j<temp;j++){
            first+=to_string(i);
        }
    }
    
    answer=first;
    answer+="0";
    reverse(first.begin(),first.end());
    answer+=first;
    
    return answer;
}

File: Java/LV1/푸드파이트 대회.java
import java.util.*;

class Solution {
    public String solution(int[] food) {
        String answer = "";
        
        for(int i=1;i<food.length;i++){
            int cur = food[i];
            
            for(int j=0;j<cur/2;j++){
                answer+=Integer.toString(i);
            }
        }
        StringBuffer sb = new StringBuffer(answer);
        String reverse = sb.reverse().toString();
        answer+="0";
        answer+=reverse;
        
        return answer;
    }
}
